compiler/Eta/Utils/Pretty.hs

```diff
@@ -6,7 +6,7 @@
 *               based on "The Design of a Pretty-printing Library"              *
 *               in Advanced Functional Programming,                             *
 *               Johan Jeuring and Erik Meijer (eds), LNCS 925                   *
-*               http://www.cs.chalmers.se/~rjmh/Papers/pretty.ps                *
+*               https://web.archive.org/web/20070203205021/http://www.cs.chalmers.se/~rjmh/Papers/pretty.ps                *
```

Comment:
Yes, this blows the ascii art boundary. I'm not certain what people would prefer to do. URL wrapping is not a good thing. Even deleting all whitespaces wouldn't result in a url that fits...